_posts/2015-11-12-Using-OCR-To-Fix-A-Hilarious-Bug.markdown

```diff
@@ -43,7 +43,7 @@ The script eventually turned into something like:
 # initialize and configure Tesseract
 engine = Tesseract::Engine.new do |config|
   config.language  = :eng
-  config.blacklist = '|'
+  config.denylist = '|'
```

Comment:
In this case, I don't think we should make this rename as [the gem doesn't currently support](https://github.com/meh/ruby-tesseract-ocr#basic-functionality-of-tesseract) this configuration option.